### PR TITLE
Fix change processing status

### PR DESCRIPTION
**Pull request by Beacze**
Depends on #1029

### CODE REVIEW

File: Kitodo/src/main/java/de/sub/goobi/persistence/apache/FolderInformation.java [hunk 1]
```diff
@@ -63,57 +63,46 @@ public URI getImagesTifDirectory(boolean useFallBack) {
...
 
-        return rueckgabe;
-    }
-
-    /**
-     * Get tif directory exists.
-     *
-     * @return true if the Tif-Image-Directory exists, false if not
-     */
-    public Boolean getTifDirectoryExists() {
```
Comment (henning-gerhardt) [inline Kitodo/src/main/java/de/sub/goobi/persistence/apache/FolderInformation.java]: You can not remove method `getTifDirectoryExists()` as this method is still used in `Kitodo/src/main/webapp/pages/inc_Prozessverwaltung/Prozesse_Liste.xhtml` lines 468, 476, 483, 490, 496 and 503!

Comment (Beacze) [reply Kitodo/src/main/java/de/sub/goobi/persistence/apache/FolderInformation.java]: item.tifDirectoryExists - item is process object, so this method should be in ProcessDTO:

`<t:dataTable id="auflistung" styleClass="standardTable" width="100%" cellspacing="1px" cellpadding="1px"
                 headerClass="standardTable_Header"
                 rowClasses="standardTable_Row1,standardTable_Row2,standardTable_Row1,standardTable_Row2" var="item"
                 value="#{ProzessverwaltungForm.page.listReload}">`

Comment (henning-gerhardt) [reply Kitodo/src/main/java/de/sub/goobi/persistence/apache/FolderInformation.java]: Then this method must be available in `ProcessDTO` which is not the case yet.

Comment (Beacze) [reply Kitodo/src/main/java/de/sub/goobi/persistence/apache/FolderInformation.java]: Actually it is:

public boolean isTifDirectoryExists() {
        return tifDirectoryExists;
    }

As it is boolean, it works with is....: https://stackoverflow.com/questions/14400222/boolean-properties-starting-with-is-does-not-work

Comment (henning-gerhardt) [reply Kitodo/src/main/java/de/sub/goobi/persistence/apache/FolderInformation.java]: Ah, ok. Thank you for this explanation. Then this method could be removed.

**Comment by henning-gerhardt** [comment]
There is a merge conflict in `Kitodo/src/main/java/de/sub/goobi/forms/ProzessverwaltungForm.java` - please resolve this conflict.

**Pull request merged**